cli/jobs/basics/hello-interactive.yml

```diff
@@ -0,0 +1,12 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandJob.schema.json
+command: echo "hello world" | sleep 3600
+environment:
+  image: azureml:AzureML-sklearn-1.0-ubuntu20.04-py38-cpu@latest
+compute: azureml:cpu-cluster
+services: 
+  my_vscode: 
+    job_service_type: VS_code 
```

Comment:
lower case "vs_code".

Reply:
yes, the values are case sensitive and should be one of the below
"jupyter_lab", "ssh", "tensor_board", "vs_code"

https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/job_service.py#:~:text=job_service_type%3A%20Optional%5BLiteral%5B%22jupyter_lab%22%2C%20%22ssh%22%2C%20%22tensor_board%22%2C%20%22vs_code%22%5D%5D%20%3D%20None%2C

Reply:
done